Makefile

```diff
@@ -578,6 +578,34 @@ kind-install-cilium-clustermesh: kind-clustermesh-ready ## Install a local Ciliu
 	$(CILIUM_CLI) clustermesh status --context kind-clustermesh1 --wait
 	$(CILIUM_CLI) clustermesh status --context kind-clustermesh2 --wait
 
+kind-install-cilium-clustermesh-fast:kind-clustermesh-ready
+	@echo "  INSTALL cilium on clustermesh1 cluster"
+	kind load docker-image --name clustermesh1 quay.io/cilium/cilium-ci:latest
+	-$(CILIUM_CLI) --context=kind-clustermesh1 uninstall >/dev/null
+	$(CILIUM_CLI) --context=kind-clustermesh1 install \
+		--chart-directory=$(ROOT_DIR)/install/kubernetes/cilium \
+		--values=$(ROOT_DIR)/contrib/testing/kind-clustermesh1.yaml \
+		--values=$(ROOT_DIR)/contrib/testing/kind-fast.yaml \
+		--set=clustermesh.apiserver.kvstoremesh.enabled=$(ENABLE_KVSTOREMESH)
+
+	@echo "  INSTALL cilium on clustermesh2 cluster"
+	kind load docker-image --name clustermesh2 quay.io/cilium/cilium-ci:latest
+	-$(CILIUM_CLI) --context=kind-clustermesh2 uninstall >/dev/null
+	$(KUBECTL) --context=kind-clustermesh1 get secret -n kube-system cilium-ca -o yaml | \
+		$(KUBECTL) --context=kind-clustermesh2 replace --force -f -
+	$(CILIUM_CLI) --context=kind-clustermesh2 install \
+		--chart-directory=$(ROOT_DIR)/install/kubernetes/cilium \
+		--values=$(ROOT_DIR)/contrib/testing/kind-clustermesh2.yaml \
+		--values=$(ROOT_DIR)/contrib/testing/kind-fast.yaml \
+		--set=clustermesh.apiserver.kvstoremesh.enabled=$(ENABLE_KVSTOREMESH)
+
+	$(MAKE) kind-image-fast
+
+	@echo "  CONNECT the two clusters"
+	$(CILIUM_CLI) clustermesh connect --context kind-clustermesh1 --destination-context kind-clustermesh2
+	$(CILIUM_CLI) clustermesh status --context kind-clustermesh1 --wait
+	$(CILIUM_CLI) clustermesh status --context kind-clustermesh2 --wait
```

Comment:
I wouldn't mind some factoring between the two targets. Maybe the `CONNECT` part, for example, could be a separate target shared by the two (“slow” and fast) recipes? Not a strong requirement, though.

Reply:
Agreed that it seems less DRY than it could be. I spent some time trying to find an angle to just supply `FAST` as an env to avoid creating a new target, but that ended up being more complex and doesn't follow the existing convention with the other `fast` target.

I think splitting at least the connect target makes sense :+1: